modules/education/src/androidTest/java/edu/stanford/spezi/modules/education/EducationModule.kt

```diff
@@ -0,0 +1,19 @@
+package edu.stanford.spezi.modules.education
+
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import edu.stanford.spezi.modules.education.videos.data.repository.EducationRepository
+import io.mockk.mockk
+
+@Module
+@InstallIn(SingletonComponent::class)
```

Comment:
Even while I use the parameterless version, Hilt complains that the module is missing. If I want to specify in the TestInstallIn what is to be replaced, I end up with the problem that the module is in the app, which is correct. However, we would actually prefer to leave the test in the module. How can we solve this elegantly?

Reply:
You can leave the replaces as empty

Also in your current `VideoScreenTest` you don't need  `@HiltAndroidTest`, `HiltAndroidRule`, and just use a `createComposeRule()`. Hilt setup is only needed in case you use hilt dependencies (view model in the parameterless composable). You can ignore for now and create a small issue for this to adjust